exampleSite/content/about-split/index.md

```diff
@@ -4,11 +4,11 @@ lastmod = "2017-10-05"
 date = "2017-10-05"
 subtitle = ""
 layout = "split"
-sidebarTitle = "Sidebar"
+sidebar_title = "Sidebar"
```

Comment:
What's the decision on nesting vs non-nesting? Or do you want to do that in a follow-up PR, if we decide for nesting?

Reply:
I think we should do a follow up on a new PR. But what's the final decision on nesting it looks good and more detailed.